css/grids.css

```diff
@@ -90,7 +86,6 @@ Song_ID2, Title2, Artist2, Album2, Genre2, Size2, Duration2 {
 	width: 400px;
 	height: 200px;
 	overflow-y: scroll;
-	float: center;
```

Comment:
validator code: wasn't consider as a value and did not work within my website, deleted the code